src/accountsdb/accounts_file.zig

```diff
@@ -253,7 +252,7 @@ pub const AccountFile = struct {
             null,
             file_size,
             std.posix.PROT.READ | std.posix.PROT.WRITE,
-            std.posix.MAP{ .TYPE = .SHARED },
+            std.posix.MAP{ .TYPE = .PRIVATE },
             file.handle,
```

Comment:
Doesn't this mean that writes aren't carried back to the file; is this intentional?

Reply:
See previous discussion on the topic: https://github.com/Syndica/sig/pull/289#discussion_r1786576053
TLDR; this is correct.

Reply:
Perfect, just double checking :ok_hand: